frontend/src/store/products/actions.js

```diff
@@ -0,0 +1,65 @@
+import { api } from "../../utils/Api"
+
+/**
+ * products list actions types
+ */
+export const PRODUCT_LIST_REQUEST = "PRODUCT_LIST_REQUEST"
```

Comment:
Why you don't define action types as constants in separate files?

Reply:
i thought there is no need for